Scripts/CreatePackages.py

```
@@ -0,0 +1,203 @@
+import sys
+import argparse
+import VersionHelper
+from pathlib import Path
+from py7zr import SevenZipFile
+import shutil
+
+SCRIPTS_DIR = Path(__file__).parent.resolve()
+ROOT_DIR = SCRIPTS_DIR.parent
+SOLUTION_FILE = ROOT_DIR / 'AltTab.sln'
+
+def find_vcvarsall(version=17.7):
```

Ruff (0.14.10)

Unused function argument: `version`

(ARG001)